src/main/scala/scala/tools/refactoring/implementations/oimports/Region.scala

```diff
@@ -0,0 +1,78 @@
+package scala.tools.refactoring
+package implementations.oimports
+
+import scala.reflect.internal.util.SourceFile
+import scala.tools.nsc.Global
+import scala.tools.refactoring.common.Change
+import scala.tools.refactoring.common.TextChange
+import scala.util.Properties
+
+case class Region private (imports: List[Global#Import], owner: Global#Symbol, startPos: Global#Position, endPos: Global#Position,
```

Comment:
not sure, I would allow to leak it. Maybe other packs could use it? If you insist I make it pack private.